R/baselinenowcast_df-class.R

```diff
@@ -72,6 +66,35 @@ new_baselinenowcast_df <- function(baselinenowcast_df,
   return(result)
 }
 
+#' Align integer time indices with reference dates
+#' @description Internal helper function that merges reference dates into a
+#'   nowcast dataframe, replacing integer time indices with actual dates.
+#'
+#' @param baselinenowcast_df Data.frame containing nowcast information with an
```

Comment:
For a separate PR I think but am wondering if we should change the input of `new_baselinenowcast_df` so that the input isn't a `baselinenowcast_df` but maybe a `raw_nowcast_df` (e.g. the output of `sample_nowcasts()`) so as not to confuse with the class object. I can make a separate issue.